common/types/beacon.go

```diff
@@ -20,19 +20,9 @@ type Beacon [BeaconSize]byte
 // EmptyBeacon is a canonical empty Beacon.
 var EmptyBeacon = Beacon{}
 
-// Hex converts a hash to a hex string.
-func (b Beacon) Hex() string { return util.Encode(b[:]) }
```

Comment:
I believe this function `util.Encode` isn't used anywhere any more outside of tests and a single error case in `sql/database.go` that could also be replaced with `hex.EncodeToString` shall we remove it as part of this PR as well?

Reply:
It's also used in common/util/json.go which I don't want to refactor here.

Reply:
The usage in `common/util/json.go` is for the `String` method of `Bytes` which is only used in tests. The `Bytes` type in general seems to be unused. It serves as tool to serialise `[]byte` types to `json` but I can't seem to find any place in our codebase where `types.Hash32` and `types.Hash20` are encoded/decoded as or from json 🤔

Reply:
I don't think it is necessary to remove this code as part of this PR, but since it appears to not actually be used maybe as part of another PR? 🙂